### PR TITLE
Update Nushell template to work with Nushell 0.89.

### PR DESCRIPTION
**Pull request by tage64**

### CODE REVIEW

**Comment by cvanelteren** [comment]
Thanks!

**Comment by happysalada** [comment]
@ajeetdsouza if possible, if after this PR was merged, you could make another release that would be great.
No worries if you are busy.
Thank you for the time you've put on this repo!

File: templates/nushell.txt [hunk 1]
```diff
@@ -44,7 +44,7 @@ def --env __zoxide_z [...rest:string] {
   let path = if (($rest | length) <= 1) and ($arg0 == '-' or ($arg0 | path expand | path type) == dir) {
     $arg0
   } else {
-    (zoxide query --exclude $env.PWD -- $rest | str trim -r -c "\n")
+    (zoxide query --exclude $env.PWD -- ...$rest | str trim -r -c "\n")
```
Comment (FrancescElies) [inline templates/nushell.txt:47]: I think the same change is missing in `__zoxide_zi`, isn't it?

Comment (FrancescElies) [reply templates/nushell.txt:47]: I just saw there is #663, exactly doing that.

**Comment by ajeetdsouza** [comment]
Closed in favor of #663.

**Pull request closed without merging**